File: Cpp/For e While/2.cpp
//Lista 2 do JUDE
//Questão 2

#include <iostream>
using namespace std;

int main() 
{
int N, P, Contador, Valor;
cin >> N;
Contador = 0;
Valor = 0;

    while (Contador < N)
        { 
        cin >> P;
            if (P > Valor)
            {
                Valor = P;
            }
        Contador = Contador + 1;
        }
cout << Valor << endl;
return 0;
}

File: Python/Matriz/2.py
#Prova 3 do JUDE
#Questão 2

linhas = 8
colunas = 8
matriz = []
for i in range(linhas):
    linha = [int(x) for x in input().split()]
    matriz.append(linha)

X, Y = map(int, input().split())
(matriz[X][Y]) = 1

contador = 0
for i in range(linhas):
    if matriz[X][i] == 2:
        contador += 1
        break
for i in range(colunas):
    if matriz[i][Y] == 2:
        contador += 1
        break
print(contador)